src/ol/Feature.js

```diff
@@ -252,7 +252,7 @@ Feature.prototype.setStyle = function(style) {
  * Set the feature id.  The feature id is considered stable and may be used when
  * requesting features or comparing identifiers returned from a remote source.
  * The feature id can be used with the
- * {@link module:ol/source/Vector~Vector#getFeatureById} method.
+ * {@link module:ol/source/Vector#getFeatureById} method.
```

Comment:
In https://github.com/openlayers/openlayers/pull/8123#discussion_r184369564 you mention that we need to use the long form for deep links like this.  Curious which will work.